libp2p/muxers/mplex/lpchannel.nim

```diff
@@ -93,12 +93,45 @@ proc resetMessage(s: LPChannel) {.async.} =
   await s.conn.writeMsg(s.id, s.resetCode)
 
 proc resetByRemote*(s: LPChannel) {.async.} =
-  await allFutures(s.close(), s.closedByRemote())
+  let
+    f1 = awaitne s.close()
+    f2 = awaitne s.closedByRemote()
```

Comment:
close serially:
```
    f1 = awaitne s.close()
    f2 = awaitne s.closedByRemote()
```

close in parallel:
```
  let
    f1 = s.close()
    f2 = s.closedByRemote()
  let
    f1done = awaitne f1
    f2done = awaitne f2
```

Reply:
or:
```
  let
    f1 = s.close()
    f2 = s.closedByRemote()
  try:
    let f1v = await f1
  except CatchableError as e:
   ...
  try:
    let f2v = await f2
  except CatchableError as e:
   ...
```

Reply:
Oops. That was the way I meant to do it, not sure why I ended up doing it sync, had my mind elsewhere probably 🤔

Anyway will use the new all finished call from now.